src/tdlibreceiver.cpp

```diff
@@ -146,7 +146,7 @@ void TDLibReceiver::setActive(bool active)
 void TDLibReceiver::receiverLoop()
 {
     LOG("Starting receiver loop");
-    const double WAIT_TIMEOUT = 5.0;
+    const double WAIT_TIMEOUT = 1.0;
```

Comment:
Was this a necessary change or just something you tried while debugging the delay and left in?
Just thinking about how this may or may not affect high latency connections…

Reply:
Definitely a leftover, reverted it with the latest commit...